src/hotspot/share/classfile/fieldLayoutBuilder.cpp

```diff
@@ -547,14 +547,17 @@ void FieldLayoutBuilder::prologue() {
 //     (support of the @Contended annotation)
 //   - @Contended annotation is ignored for static fields
 void FieldLayoutBuilder::regular_field_sorting() {
-  for (AllFieldStream fs(_fields, _constant_pool); !fs.done(); fs.next()) {
+  int idx = 0;
+  for (GrowableArrayIterator<FieldInfo> it = _field_info->begin(); it != _field_info->end(); ++it, ++idx) {
+    FieldInfo ctrl = _field_info->at(0);
     FieldGroup* group = nullptr;
-    if (fs.access_flags().is_static()) {
+    FieldInfo tfi = *it;
```

Comment:
What's the 't' in tfi?  Maybe a longer variable name would be helpful here.

Reply:
At some point there was a TempFieldInfo type, hence the name.
Renamed to fieldinfo.